src/main/java/apoc/export/cypher/ExportFileManager.java

```diff
@@ -12,4 +12,6 @@
     String drain(String type);
 
     String getFileName();
+
+    Boolean separatedFiles();
```

Comment:
what is meant for? I cannot find any info in the documentation can you please add it?